slim/export_inference_graph.py

```diff
@@ -107,7 +112,10 @@ def main(_):
         num_classes=(dataset.num_classes - FLAGS.labels_offset),
         is_training=FLAGS.is_training)
     if hasattr(network_fn, 'default_image_size'):
-      image_size = network_fn.default_image_size
+      if FLAGS.override_default_image_size:
+        image_size = FLAGS.default_image_size
+      else:
+        image_size = network_fn.default_image_size
```

Comment:
I am OK with both ways. Using something like the eval_image_size flag and changing the FLAGS.default_image_size will change more files, so I didn't do it.